app/src/main/AndroidManifest.xml

```diff
@@ -16,8 +18,18 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
-        <activity android:name=".BannersActivity"
+        <activity
+            android:name=".BannersActivity"
             android:label="Banners" />
+
+        <activity android:name=".ProfileActivity">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
```

Comment:
Does this have to be `MAIN`? Can we instead get to this activity from a button in the `MainActivity`?